src/Components/Reports/Common/ExecutiveFirstPage.js

```diff
@@ -7,15 +7,14 @@ import styles from './styles';
 import { IconCanvas } from './IconCanvas';
 
 export const ExecutiveFirstPage = ({ data }) => {
-    const { conditions_count: conditionsCount, non_optimized_count: nonOptimizedCount, total_count: totalCount } = data?.meta;
+    const { conditions_count: conditionsCount, non_optimized_count: nonOptimizedCount,
+        total_count: totalCount, stale_count: staleCount } = data?.meta;
     const optimizedCount = data?.systems_per_state?.optimized?.count;   /* eslint-disable-line camelcase */
     const newLine = '\n';
-    const bullet = '\u2022';
 
     const ioOccuranceTableData = [
         [
-            <View key={'disk-io-title'} style={{ display: 'flex',
-                flexDirection: 'row' }}>
+            <View key={'disk-io-title'} style={styles.flexRow}>
```

Comment:
@PreetiW  - I see repetitive pattern here while declaring `ioOccuranceTableData,  ramOccuranceTableData, cpuOccuranceTableData` and then you are pushing extra occurances. Will it be good to extract out this in a method and pass table specific data per section and `occurrences` at the same time?

Reply:
nitpick - correct the typo from `occurances` to `occurrences` at [lines](https://github.com/RedHatInsights/ros-frontend/pull/136/files#diff-b7de4f9ad1237bb1ab20430045fd725531460730df91ff77373ba8baf08adaa0R49-R51)

Reply:
@kgaikwad regarding your first comment can you please share the example for same? the first declaration is around adding the table name and later adding the data. The data generation part is already part of separate Util method.

Reply:
> nitpick - correct the typo from `occurances` to `occurrences` at [lines](https://github.com/RedHatInsights/ros-frontend/pull/136/files#diff-b7de4f9ad1237bb1ab20430045fd725531460730df91ff77373ba8baf08adaa0R49-R51)

good catch @kgaikwad fixed it and will push the changes in next commit. 💯

Reply:
Need to extract the creation logic for occurrences and move into the Util method.

Reply:
done @kgaikwad